### DISCUSSION TITLE
Unable to reset password upon receiving password reset link

### DISCUSSION THREAD

**Discussion by nmikovic** [community/community#129167] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question
How to recover user account
### Body

![image](https://github.com/community/community/assets/10166201/2112be54-dbec-4924-abeb-de928a9f8d5b)

Upon trying to reset password on one of my accounts, I'm getting this email but recovery code is not sent to my phone/email.

![image](https://github.com/community/community/assets/10166201/0242c0f2-214c-48b3-b668-deb832276759)

What to do as I need to recover this account for business purpose

**Comment by nmikovic** [comment]
Ok, after some time got sms bomb, resolved

**Reply by ebndev** [reply]
Hi @nmikovic, I'm glad you were able to get your issue resolved. I will go ahead and close this Discussion.

In case this may be a better solution for the future, you can also enable 2FA using a [TOTP app](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

**Comment by Rudra-Sankha-Sinhamahapatra** [comment]
i think you tried a very high times password reseting..and github sent you many messages..thats why redis had rate limited your message on queue
